client/src/components/ReviewTab/ReviewTab.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Typography, Grid } from '@material-ui/core';
+import ReviewTabForm from './ReviewTabForm/ReviewTabForm';
+import { Review } from '../../interface/User';
+import useStyles from './useStyles';
+import ArtistCarousel from '../ArtistCarousel/ArtistCarousel';
+import Switch from '@material-ui/core/Switch';
+import { ChangeEvent, useState } from 'react';
+
+export default function PersonalInfo(): JSX.Element {
+  const classes = useStyles();
+  const [checked, setChecked] = useState(false);
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setChecked(event.target.checked);
+  };
+
+  const handleSubmit = ({ rating, text }: Review) => {
```

Comment:
Couple things here. I'm assuming this is supposed to make an API call? haha. Also don't forget to take in the event and prevent the default if this is going to be called on submit of a form.

Reply:
yes for calling the api; sure will do that ; thanks :)